input/resources/au-core-server.xml

```diff
@@ -3029,7 +3029,7 @@
 <title value="AU Core Server CapabilityStatement"/>
 <status value="draft"/>
 <date value="2023-05-15"/>
-<description value="This CapabilityStatement describes the basic rules for the AU Core server actor that is responsible for providing responses to queries submitted by AU Core clients. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU Core servers are defined in this CapabilityStatement."/>
```

Comment:
The change request is to change au-core-client NOT au-core-server. Still need to apply the approved changes of https://jira.hl7.org/browse/FHIR-45112.

Reply:
Updated AU Core Client description.